clients/tests/src/main/java/org/projectnessie/client/tests/AbstractSparkTest.java

```diff
@@ -32,32 +31,38 @@
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.io.TempDir;
 import org.projectnessie.client.NessieClient;
 
 public abstract class AbstractSparkTest {
   private static final Object ANY = new Object();
+
+  @TempDir File tempFile;
+
   private static final int NESSIE_PORT = Integer.getInteger("quarkus.http.test-port", 19121);
   protected static SparkConf conf = new SparkConf();
 
   protected static SparkSession spark;
-  protected static Configuration hadoopConfig = new Configuration();
   protected static String url = String.format("http://localhost:%d/api/v1", NESSIE_PORT);
 
   protected static NessieClient nessieClient;
 
   @BeforeEach
   protected void create() throws IOException {
-    String branch = "main";
-    String authType = "NONE";
+    Map<String, String> nessieParams =
+        ImmutableMap.of("ref", "main", "uri", url, "warehouse", tempFile.toURI().toString());
 
-    hadoopConfig.set(CONF_NESSIE_URI, url);
-    hadoopConfig.set(CONF_NESSIE_REF, branch);
-    hadoopConfig.set(CONF_NESSIE_AUTH_TYPE, authType);
+    nessieParams.forEach(
```

Comment:
Can you add something to `NessieConfigConstants` to avoid having these "duplicate config-key setup code" for Delta & Iceberg? Or maybe at least have the "short config keys" (like "ref" and "uri") in `NessieConfigConstants`? Guess, the latter could be useful.

Reply:
I have a separate branch where I remove the `nessie.` prefix from the constants in `NessieConfigConstants`. If you don't mind I would like to do it as a separate PR. 

No client that I am aware of uses the `nessie.` prefix on those configs and they are redundant and cause everyone to strip out the prefix.

Reply:
WFM